crates/orderbook/src/fee.rs

```diff
@@ -96,13 +97,23 @@ pub struct FeeData {
 }
 
 /// Everything required to compute the fee amount in sell token
-#[derive(Debug, Default, Clone, Copy, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq)]
 pub struct FeeParameters {
     pub gas_amount: f64,
     pub gas_price: f64,
     pub sell_token_price: f64,
 }
 
+impl Default for FeeParameters {
+    fn default() -> Self {
+        Self {
+            gas_amount: 0.,
+            gas_price: 0.,
+            sell_token_price: 1.,
```

Comment:
Just to make sure I understand the change around `Default` right:
When we have to return a `FeeParameter` for a partially fillable order it doesn't really matter which exact `sell_token_price` gets used because we also specify a fee amount of `0`. As long as `sell_token_price` is not `0` (to avoid division by 0), we are fine?
If that's not the case I'm not sure how you arrived at `1.0` for the price.

Reply:
This is correct. I just thought that the multiplicative identity made sense for the price. I will add a comment.